src/theme/colors.ts

```diff
@@ -1,15 +1,215 @@
+import { border, hexToRgb } from '@mui/system';
+
 export const colorTextPrimary = '#17324D';
 export const shadowColor = '#002B55';
 export const backdropBackground = '#17324D';
 export const menuItemBackground = '#17324D';
 export const colorPrimaryContainedHover = '#0055AA'; // Not exposed by the theme object
+
+// base palette
+export const neutral = {
```

Comment:
Maybe we can merge all the base object colors (i.e. neutral, blue ecc...) into an object named palette.
Let me explain better.
I should avoid to export all these const, but i should let this variables internal (without export key) and I should define a palette const that includes all these variables and I should export it.
Here an example:
```
const neutral = {...};
const blue = {...};
const red = {...}
...

export con palette = {
 neutral,
 red,
 blue,
 ...
}
```